runtime/metricsview/executor_rewrite_approx_comparisons.go

```diff
@@ -46,6 +46,12 @@ func (e *Executor) rewriteApproxComparisonNode(a *AST, n *SelectNode) bool {
 	}
 	sortField := a.Root.OrderBy[0]
 
+	// if there are unnests in the query, we can't rewrite the query
+	// for example druid fails with join on cte having multi value dimension, issue - https://github.com/apache/druid/issues/16896
+	if len(a.unnests) > 0 {
+		return false
+	}
```

Comment:
Actually, should it be something like `if e.olap.Dialect == drivers.DialectDruid && len(a.unnests) > 0`?